src/Nancy.Bootstrappers.StructureMap.Tests/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="structuremap" version="3.1.5.154" targetFramework="net40" />
-  <package id="xunit" version="1.9.1" targetFramework="net40" />
-  <package id="xunit.extensions" version="1.9.1" targetFramework="net40" />
+  <package id="structuremap" version="3.1.6.186" targetFramework="net4" />

```

Comment:
Nevermind. Looks like it is; https://github.com/search?q=net4+extension%3Aconfig